src/core_atmosphere/mpas_atm_core_interface.F

```diff
@@ -134,6 +135,14 @@ function atm_setup_packages(configs, packages, iocontext) result(ierr)
          iauActive = .false.
       end if
 
+      nullify(config_jedi_da)
+      call mpas_pool_get_config(configs, 'config_jedi_da', config_jedi_da)
+
+      nullify(jedi_daActive)
+      call mpas_pool_get_package(packages, 'jedi_daActive', jedi_daActive)
+
+      jedi_daActive = config_jedi_da
+
```

Comment:
It's not at all mandatory, but in case we'd like to be really thorough here (I know we didn't do this for IAU, which is my fault), we could set `jedi_daActive` only if both `jedi_daActive` and `config_jedi_da` are associated. With pools, we pass in a pointer, requesting that it be set to point to the named entity; but, if that entity doesn't exist for some reason (e.g., because someone accidentally removed it from the `Registry.xml` file), the pointer is left unchanged, which in this case would leave the pointer as unassociated. So, the code could look like this:
```
      if (associated(config_jedi_da) .and. associated(jedi_daActive)) then
         jedi_daActive = config_jedi_da
      else
         ierr = ierr + 1
         call mpas_log_write('Package setup failed for ''jedi_da''. '// &
              'Either ''jedi_da'' is not a package, or ''config_jedi_da'' is not a namelist option.', &
              messageType=MPAS_LOG_ERR)
      end if
```

Reply:
Actually, even with this extra logic, there's still the issue that the physics package setup further down will simply overwrite the value of `ierr` before the routine returns. If you're willing to add some extra logic for the `jedi_da` package, I'll push an additional commit to this branch before merging to address the physics package setup problem.

Reply:
If by "add some extra logic for the `jedi_da` package", you mean insert the code above, then I am very willing.  Is there anything else you need me to change?

Reply:
Sorry, I was a little unclear. If you wouldn't mind committing the code section from my comment to check that both `jedi_daActive` and `config_jedi_da` are both associated, and to print an error if not, that would be excellent. I'll then push a commit to address the potential issue of the call to `atmphys_setup_packages` overwriting any non-zero value of `ierr`. Then I think we're all set for this PR.

Reply:
Ok, all set.